### ISSUE TITLE
Pagination isn't accessible

### ISSUE THREAD

**Issue by bethcollins92** [canonical-web-and-design/vanilla-framework#4256]
Accordiong to accessibility guidelines, the pagination component should include: 

- `aria-current="true"` to the active page
- `aria-label="page [x]"` to each link
- and it should be wrapped in a nav element with the `aria-label="pagination"`

https://tink.uk/using-the-aria-current-attribute/
https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-pagination
